src/modules/order/service.ts

```diff
@@ -0,0 +1,29 @@
+import Order from '@/modules/order/index';
+import { OrderType } from '@/modules/order/type';
+
+export class OrderService {
+  public orderList: Order[] = [];
+
+  public add(order: Order) {
+    if (this.orderList.length === 0) this.orderList.push(order);
+    else this.orderList = [...this.orderList, order];
```

Comment:
```suggestion
   this.orderList = [...this.orderList, order];
```
이 코드와 같은 동작을 하지 않을까요~? 😮